llvm/test/tools/obj2yaml/XCOFF/aux-aligntype.yaml

```diff
@@ -0,0 +1,116 @@
+## Check that obj2yaml can parse SymbolAlignmentAndType, SymbolAlignment,
+## and SymbolType.  If either SymbolAlignment or SymbolType are specified
```

Comment:
The last sentence of this is not needed up front and should be with the specific case where it's relevant (you already have it there).